packages/xstate-angular/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@xstate/angular",
+  "version": "3.1.1",
+  "description": "XState tools for Vue",
+  "keywords": [
+    "state",
+    "machine",
+    "statechart",
+    "scxml",
+    "state",
+    "graph",
+    "angular"
+  ],
+  "type": "module",
+  "author": "David Khourshid <davidkpiano@gmail.com>",
+  "homepage": "https://github.com/statelyai/xstate/tree/main/packages/xstate-angular#readme",
+  "license": "MIT",
+  "main": "dist/xstate-angular.cjs.js",
+  "module": "dist/xstate-angular.esm.js",
+  "exports": {
+    ".": {
+      "types": {
+        "import": "./dist/xstate-angular.cjs.mjs",
+        "default": "./dist/xstate-angular.cjs.js"
+      },
+      "development": {
+        "module": "./dist/xstate-angular.development.esm.js",
+        "import": "./dist/xstate-angular.development.cjs.mjs",
+        "default": "./dist/xstate-angular.development.cjs.js"
+      },
+      "module": "./dist/xstate-angular.esm.js",
+      "import": "./dist/xstate-angular.cjs.mjs",
+      "default": "./dist/xstate-angular.cjs.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "imports": {
+    "#is-development": {
+      "development": "./src/true.ts",
+      "default": "./src/false.ts"
+    }
+  },
+  "sideEffects": false,
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/statelyai/xstate.git"
+  },
+  "bugs": {
+    "url": "https://github.com/statelyai/xstate/issues"
+  },
+  "peerDependencies": {
+    "@angular/core": "^17.0.0",
+    "xstate": "^5.9.1"
+  },
+  "peerDependenciesMeta": {
+    "xstate": {
+      "optional": true
+    }
+  },
+  "devDependencies": {
+    "@angular/core": "^17.0.0",
+    "@testing-library/angular": "15.2.0",
+    "@testing-library/jest-dom": "^6.1.4",
+    "ng-packagr": "^17.0.0",
```

Comment:
How strongly required this is? it might be slightly challenging to incorporate this into our current setup. It's certainly doable but if we wouldn't have to use this... that would be kinda simpler for us

Reply:
I think it is required but I will check in the morning

Reply:
yeah it is needed. Not sure what the best way to move forward is on this one, as using babel/preconstruct doesn't make a whole lot of sense here (from what I can say)

Reply:
I can figure it out on my own before we land this - I don't want to put on you the burden of figuring intricacies of our build pipeline. I wonder, what makes it *required*? Angular surely has to be able to consume regular npm packages - what's different about a package like this one?

Reply:
https://angular.io/guide/angular-package-format They are doing pre-optimizations, it would probably work fine. 
For now I will just exclude it from preconstruct.

We can also schedule a call and figure out the build together, also feel somewhat bad to put that burden on you :D